va/va.h

```diff
@@ -3722,6 +3722,21 @@ VAStatus vaSyncSurface (
     VASurfaceID render_target
 );
 
+/**
+ * This function blocks during specified timeout (in milliseconds) until
+ * all pending operations on the render target have been completed.
+ * If timeout is zero, the function returns immediately.
+ * Return values:
+ * VA_STATUS_SUCCESS - all pending operations completed successfully;
+ * VA_STATUS_ERROR_HW_BUSY - operations are still in progress;
+ * the other error statuses indicating that operations failed;
+ */
+VAStatus vaSyncSurface2 (
+    VADisplay dpy,
+    VASurfaceID render_target,
+    unsigned int timeout
```

Comment:
better to use timeout_ms or timeout_ns to emphasize values...

Reply:
Okay.. Thanks!